syntax = "proto3";

package game_download;

service GameDownload {
    rpc DownloadLink (LinkRequest) returns (LinkResponse) {}
}

message LinkRequest {
    string download_key = 1;
}

message LinkResponse {
    string message = 1;
}